xbmc/cores/RetroPlayer/cheevos/Cheevos.h

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright (C) 2020-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "RConsoleIDs.h"
+
+#include <map>
+#include <string>
+
+namespace KODI
+{
+namespace GAME
+{
+class CGameClient;
+}
+
+namespace RETRO
+{
+class CCheevos
+{
+public:
+  CCheevos(GAME::CGameClient* gameClient, const std::string userName, const std::string loginToken);
+  void ResetRuntime();
+  void EnableRichPresence();
+  std::string GetRichPresenceEvaluation();
+
+private:
+  bool LoadData();
+  int ConsoleID();
+
+  GAME::CGameClient* const m_gameClient;
+  std::string m_userName;
+  std::string m_loginToken;
+  std::string m_romHash{};
+  std::string m_richPresenceScript{};
```

Comment:
Remove the {} on these two lines:

```c++
  std::string m_romHash;
  std::string m_richPresenceScript;
```

Because std::string has a constructor, you don't need an explicit initializer like you do with the types on the next three lines.